src/java/src/test/java/com/dnanexus/DXProjectTest.java

```diff
@@ -49,6 +54,40 @@ public void tearDown() {
         }
     }
 
+    private DXFile createMinimalFile(String name) {
```

Comment:
Can you rename this fn to `uploadSmallFile()` as `create` implies only calling `file-xxxx/new`.